.github/workflows/build-images-release.yaml

```diff
@@ -6,8 +6,7 @@ on:
     branches:
       - main
     tags:
-      - v[0-9]+.[0-9]+.[0-9]+
-      - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
Don't we want a guarantee that it always has the prefix: `v[0-9]+..*`?

Reply:
yes to some extent i'm relying on maintainers to review the tag before approving the workflow run. i don't think it's worth our time to come up with a regex to exactly match semvers.